.github/workflows/build.yml

```diff
@@ -113,9 +113,9 @@ jobs:
           ls -la seedsigner-os/images
 
       - name: upload images
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4.3.3
         with:
-          name: seedsigner_os_images
+          name: seedsigner_os_images_${{ matrix.target }}
```

Comment:
due to API change, each upload from the matrix (Python 3.10, 3.12) requires a different name